Sources/PackageCollections/Model/Collection.swift

```diff
@@ -71,52 +71,57 @@ extension PackageCollectionsModel {
 
 extension PackageCollectionsModel {
     /// Represents the source of a `PackageCollection`
-    public enum PackageCollectionSource: Equatable {
-        /// Package feed at URL
-        case feed(URL)
+    public struct CollectionSource: Equatable, Hashable, Codable {
+        /// Source type
+        public let type: CollectionSourceType
+
+        /// URL of the source file
+        public let url: URL
```

Comment:
Can we add a public initialiser for this type? This will be needed if a client has to instantiate a `CollectionSource` (e.g. for adding it).

```suggestion
        public let url: URL
        
        public init(type: PackageCollectionsModel.CollectionSourceType = .feed, url: URL) {
              self.type = type
             self.url = url
         }
```

Reply:
good call